__init__.py

```diff
@@ -375,29 +441,39 @@ def upgrade_changed_props():
     SM64_ObjectProperties.upgrade_changed_props()
 
 
+def upgrade_scene_props_node():
+    """update f3d materials with SceneProperties node"""
+    has_old_f3d_mats = bool(
+        len([mat for mat in bpy.data.materials if mat.is_f3d and mat.mat_ver < MatUpdateConvert.version])
+    )
+    if has_old_f3d_mats:
+        bpy.ops.dialog.upgrade_f3d_materials("INVOKE_DEFAULT")
+
+
 @bpy.app.handlers.persistent
 def after_load(_a, _b):
+    # link_f3d_material_library()
     upgrade_changed_props()
+    upgrade_scene_props_node()
+    resync_scene_props()
 
 
 # called on add-on enabling
 # register operators and panels here
 # append menu layout drawing function to an existing window
 def register():
 
-    if bpy.app.version >= (3, 1, 0):
+    if bpy.app.version <= (3, 1, 0):
         msg = "\n".join(
             (
-                "This version of Fast64 does not work properly in Blender 3.1.0 and later Blender versions.",
+                "This version of Fast64 does not work properly in Blender 3.1.0 and earlier Blender versions.",
```

Comment:
I think changing "blender" to 3.2.0 in the bl_info is enough, it may nit prevent the addon from enabling but it will show a warning?

Reply:
hmmm well i think in this case we still don't want people using earlier versions since it would completely break. i think its nice to have a message like this